common/djangoapps/third_party_auth/tests/testutil.py

```diff
@@ -305,3 +308,70 @@ def simulate_running_pipeline(pipeline_target, backend, email=None, fullname=Non
     finally:
         pipeline_get.stop()
         pipeline_running.stop()
+
+
+class GenerateUsernameTestCase(TestCase):
```

Comment:
The `testutil.py` file is meant to contain helper methods and classes to be used in other tests, but it's not meant to contain actual test cases itself.

We should create a new `common/djangoapps/third_party_auth/tests/test_utils.py` file and move this test into it.